### ISSUE TITLE
bug: IAM Load Balancer Role lacking elasticloadbalancing:AddTags

### ISSUE THREAD

**Issue by ccoffman4477** [lablabs/terraform-aws-eks-load-balancer-controller#14]
### Summary

small change necessary in the policy document to avoid ELB provisioning errors due to lack of access (single line update for iam.tf is all that is required to address the problem)

### Issue Type

Bug Report

### Terraform Version

```console
[ec2-user@ip-10-10-101-78 ~]$ terraform --version
Terraform v1.5.5
on linux_386

[ec2-user@ip-10-10-101-78 ~]$ helm list -A
NAME                    NAMESPACE               REVISION        UPDATED                                 STATUS          CHART                                   APP VERSION
aws-ebs-csi-helm        aws-ebs-csi-helm        1               2023-08-30 15:39:52.787566047 +0000 UTC deployed        aws-ebs-csi-driver-2.10.1               1.11.2
aws-lb-controller       aws-lb-controller       1               2023-08-30 15:39:52.952624315 +0000 UTC deployed        aws-load-balancer-controller-1.4.2      v2.4.2
blueprint-staging       blueprint-staging       1               2023-08-31 17:45:04.015494376 +0000 UTC deployed        blueprint-helm-1.5.2-f7a6bce.1
cert-manager-helm       cert-manager-helm       1               2023-08-31 15:05:50.874386206 +0000 UTC deployed        cert-manager-v1.9.1                     v1.9.1
cluster-autoscaler      cluster-autoscaler      1               2023-08-30 15:36:04.251070664 +0000 UTC deployed        cluster-autoscaler-9.19.1               1.23.0
ingress-nginx-helm      ingress-nginx-helm      2               2023-08-31 16:45:21.354035999 +0000 UTC deployed        ingress-nginx-4.1.4                     1.2.1
```

### Steps to Reproduce

fails consistently 

```console
module "eks-load-balancer-controller_helm" {
  source  = "lablabs/eks-load-balancer-controller/aws"
  version = "1.1.0"

  enabled = var.load_balancer_controller_enabled

  irsa_role_name_prefix            = var.cluster_name
  cluster_name                     = var.cluster_name
  cluster_identity_oidc_issuer     = var.oidc_provider
  cluster_identity_oidc_issuer_arn = var.oidc_provider_arn

  # insert the 3 required variables here
  helm_release_name = "aws-lb-controller"
  namespace         = "aws-lb-controller"
  
  values = yamlencode({
    "podLabels" : {
      "app" : "aws-lb-controller"
    }
  })

  helm_timeout = 240
  helm_wait    = true
}
```

### Expected Results

expecting not to modify the IAM aws-lb-controller role by hand each time I make use of the Terraform module

### Actual Results

prepares the service linked role as specified in the policy document but am finding messages within the service logs indicating IAM access is not sufficient to enable the controller

**Comment by jaygridley** [comment]
Hello @ccoffman4477, could you please share the error you are getting on the AWS side? Thanks :-)

**Comment by ccoffman4477** [comment]
@jaygridley 

User: arn:aws:sts::019098429742:assumed-role/staging-v2-aws-lb-controller/1693418749026622762 is not authorized to perform: elasticloadbalancing:AddTags on resource: arn:aws:elasticloadbalancing:us-east-1:019098429742:targetgroup/k8s-ingressn-ingressn-2678d20407/* because no identity-based policy allows the elasticloadbalancing:AddTags action

**Comment by jaygridley** [comment]
@ccoffman4477 I think we might already have a fix for this in https://github.com/lablabs/terraform-aws-eks-load-balancer-controller/releases/tag/v1.2.0. Could you have a try?

**Comment by ccoffman4477** [comment]
@jaygridley confirmed, IAM issue present in v1.1.0 was resolved with v1.2.0 update to the policy document
